converters/audio_to_audio/audio_converter.py

```diff
@@ -2,39 +2,45 @@
 import ffmpeg
 from converters.converter import Converter
 
+from converters.audio_to_audio.audio_exception import AudioConversionError
+from converters.audio_to_audio.audio_options import AudioOptions
+
+
 class AudioConverter(Converter):
-    def __init__(self,audio_path):
-        self.audio_path=audio_path
+    def __init__(self, audio_path):
+        self.audio_path = audio_path
 
     def convert(self, output_format='mp3', **kwargs):
-        output_file = os.path.splitext(os.path.basename(self.audio_path))[0] +'.' + output_format
-        output_path = os.path.join('outputs','audio_converted_outputs',output_file)
-        # si el archivo ya existe no convertir
+        output_path = self._get_output_path(output_format)
+
+        # If the file already exists do not convert
         if os.path.exists(output_path):
```

Comment:
Allow user to rewrite the file even if the output file format already exists. The user may want to convert the same audio with the same format but with diferents parameters (different speed for example).